lib/publiccloud/ec2.pm

```diff
@@ -138,9 +162,9 @@ sub upload_img {
           . "--private-key-file " . $self->ssh_key_file . " "
           . "-d 'OpenQA upload image' "
           . "--wait-count 3 "
+          . "--ec2-ami '" . $helper_ami_id. "' "
```

Comment:
Why do you need this change?

Reply:
because from now on AMI is obligatory you can not skip it , because there is no `ec2utils.conf`

Reply:
Ok I see, but I would just keep the name `$ami_id`. Or at least use the same name everywhere in this function.

Reply:
I just thought that `ami_id` is confusing because uploaded image will also have AMi id

Reply:
fine for me